node/lib/util/cherry_pick_util.js

```diff
@@ -704,15 +704,17 @@ exports.closeSubs = co.wrap(function *(opener, changes) {
  *
```

Comment:
The change looks good.  If you can, please "squash" the two commits into one.  Generally for small changes like this it's neater if they are made using one commit.